modules/Bio/EnsEMBL/IO/Writer/ColumnBasedGeneric.pm

```diff
@@ -175,7 +175,7 @@ sub combine_fields {
 	    }
     }
 
-    return join $delimiter, @values;
+    return join($delimiter, @values);
```

Comment:
Come on, even Conway doesn't demand these brackets.

Reply:
lol
was not on purpose, was doing something else and later discarded (that change must fave slipped)